core/domain/user_services_test.py

```diff
@@ -642,6 +819,88 @@ def test_get_user_ids_by_role(self):
                 feconf.ROLE_ID_CURRICULUM_ADMIN)),
             set([user_ids[2], user_ids[3]]))
 
+    def test_update_user_bio(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+        user_bio = 'new bio'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_services.update_user_bio(user_id, user_bio)
+        user_settings = user_services.get_user_settings_from_email(user_email)
+
+        self.assertEqual(user_bio, user_settings.user_bio)
+
+    def test_update_preferred_language_codes(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_settings_model = user_models.UserSettingsModel.get_by_id(user_id)
+        user_services.update_preferred_language_codes(
+            user_id, user_settings_model.preferred_language_codes)
+        user_settings = user_services.get_user_settings_from_email(user_email)
+
+        self.assertEqual(
+            user_settings_model.preferred_language_codes,
+            user_settings.preferred_language_codes
+        )
+
+    def test_update_preferred_site_language_code(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+        preferred_site_language_code = 'en'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_services.update_preferred_site_language_code(
+            user_id, preferred_site_language_code)
+        user_settings = user_services.get_user_settings_from_email(user_email)
+
+        self.assertEqual(
+            preferred_site_language_code,
+            user_settings.preferred_site_language_code
+        )
+
+    def test_update_preferred_audio_language_code(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        user_settings_model = user_models.UserSettingsModel.get_by_id(user_id)
+        user_services.update_preferred_audio_language_code(
```

Comment:
Same thing. Ditto below (in the next test). Try not to use user_settings_model in tests which only relate to the domain layer.